projects/core/src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -940,6 +940,32 @@ function urlEncode(str)
     return str
 end
 
+--- Splits a string in a table.
+--
+-- @tparam string string_to_split The string to split.
+-- @tparam string pattern At what pattern to split the string.
+-- @treturn table A table containing the splitted strings.
+-- @usage args = textutils.splitString("arg1 arg2", " ")
+-- @since edit this if it goes live (I hope it does)
+local function split(string_to_split, pattern)
+    local Table = {}
+    local fpat = "(.-)" .. pattern
+    local last_end = 1
+    local s, e, cap = string_to_split:find(fpat, 1)
+    while s do
+        if s ~= 1 or cap ~= "" then
+            table.insert(Table, cap)
+        end
+        last_end = e + 1
+        s, e, cap = string_to_split:find(fpat, last_end)
+    end
+    if last_end <= #string_to_split then
+        cap = string_to_split:sub(last_end)
+        table.insert(Table, cap)
+    end
+    return Table
+end
```

Comment:
I would recommend a much simpler split function.

[From StackOverflow](https://stackoverflow.com/questions/1426954/split-string-in-lua):

```suggestion
function split(inputstr, sep)
    expect(1, inputstr, "string")
    expect(2, sep, "string", "nil")
    if sep == nil then
        sep = "%s"
    end
    local t = {}

    for str in string.gmatch(inputstr, "([^" .. sep .. "]+)") do
        table.insert(t, str)
    end

    return t
end
```

Reply:
Alright, thanks 👍